taxonomy/models.py

```diff
@@ -0,0 +1,37 @@
+"""Models for lore taxonomy application"""
+
+from django.db import models
+
+from learningresources.models import (
+    Repository,
+    LearningResourceType,
+    LearningResource,
+)
+
+
+class Vocabulary(models.Model):
+    """Model for vocabulary table"""
+    repository = models.ForeignKey(Repository)
+    name = models.TextField()
+    description = models.TextField()
+    required = models.BooleanField()
+    vocabulary_type = models.CharField(max_length=1, choices=(
+        ("m", "managed"),
+        ("f", "free tagging")
+    ))
+    weight = models.IntegerField()
+
+    learning_resource_types = models.ManyToManyField(
+        LearningResourceType,
+        related_name="vocabularies"
+    )
+
+
+class Term(models.Model):
+    """Model for term table"""
+    vocabulary = models.ForeignKey(Vocabulary)
+    label = models.TextField()
```

Comment:
Probably another use for `unique_together`, if the label can exist for multiple vocabularies, but not multiple times for the same vocabulary.